src/ott/solvers/nn/neuraldual.py

```diff
@@ -97,6 +101,12 @@ class W2NeuralDual:
     parallel_updates: Update :math:`f` and :math:`g` at the same time
     init_f_params: initial parameters for :math:`f`
     init_g_params: initial parameters for :math:`g`
+    tau_a: If `< 1`, defines how much unbalanced the problem is
```

Comment:
In the above documentation, right after `:attr:~ott.solvers.nn.models.ModelBase.is_potential.`, I'd mention your approach for unbalancedness + cite the relevant papers.